assignments/arrays.js

```diff
@@ -66,31 +66,71 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
 
 
+for (let i = 0; i < inventory.length; i++) {
+	console.log(`Car ${inventory[i]["id"]} is a ${inventory[i]["car_year"]} ${inventory[i]["car_make"]} ${inventory[i]["car_model"]}`);
+}
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
 let lastCar = 0;
 console.log();
 
+let lastCar = 0;
+lastCar = inventory[inventory.length - 1]["car_make"] + " " + inventory[inventory.length - 1]["car_model"];
+console.log(lastCar);
+
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
 console.log();
 
+let carModels = [];
+for (let i = 0; i < inventory.length; i++) {
+	carModels.push(inventory[i]["car_model"]);
+}
+carModels.sort();
+for (let i = 0; i < carModels.length; i++) {
```

Comment:
I see what you have done here, pretty slick, I've seen majority of people just console.log(carModels) after .sort()